prow/plugins.yaml

```diff
@@ -145,9 +145,6 @@ blunderbuss:
 
 cat:
   key_path: /etc/cat-api/api-key
-heart:
-  adorees:
-  - k8s-merge-robot
```

Comment:
If we are removing all configuration for the plugin we should also disable the plugin (it is currently enabled on multiple orgs).

Reply:
Won't disabling the plugin also disable adding emojis to PRs contains OWNERS changes?